src/App.js

```diff
@@ -1,13 +1,79 @@
 import React from 'react';
+import Button from "./components/Button";
+import Product from "./components/Product";
+import Tile from "./components/Tile";
+import bag1 from './assets/bag_1.png';
+import bag2 from './assets/bag_2.png';
+import bag3 from './assets/bag_3.png';
+import bag4 from './assets/bag_4.png';
+import brand from './assets/brand.png';
+import our_story from './assets/our_story.png';
 import './App.css';
 
 function App() {
   return (
+      <>
       <h1>Handbags & Purses</h1>
+      <nav>
+          <Button title={"to the collection"} disabled={false}/>
+          <Button title={"shop all bags"} disabled={false}/>
+          <Button title={"pre-orders"} disabled={true}/>
+      </nav>
+          <main>
+              <Product
+                  title={"Best seller"}
+                  image={bag1}
+                  description={"The handy bag"}
```

Comment:
Je zou ervoor kunnen kiezen om alleen het type tas mee te geven. Omdat dat eigenlijk het enige is dat verandert. Dan kun je in je component zoiets zetten als `<p>The {description} bag</p>`